pygeo/parameterization/DVGeoMulti.py

```diff
@@ -184,6 +184,7 @@ def addIntersection(
         project=False,
         marchDir=1,
         includeCurves=False,
+        slidingCurves=[],
```

Comment:
To avoid mutable defaults, this should be initialized to None and then set to empty in `__init__` like `featureCurves`